src/main/java/org/ays/emergency_application/model/entity/EmergencyEvacuationApplicationEntity.java

```diff
@@ -92,13 +93,12 @@ public class EmergencyEvacuationApplicationEntity extends BaseEntity {
     @Column(name = "IS_IN_PERSON")
     private Boolean isInPerson;
 
-    @Column(name = "HAS_OBSTACLE_PERSON_EXIST")
-    private Boolean hasObstaclePersonExist;
+    @Column(name = "HAS_OBSTACLE_PERSON_EXIST", nullable = false)
+    private Boolean hasObstaclePersonExist = false;
```

Comment:
Burayı neden değiştirmiştik?
```suggestion
    @Column(name = "HAS_OBSTACLE_PERSON_EXIST")
    private Boolean hasObstaclePersonExist;
```

Reply:
@m1erla bu issue'nun kapsamında değil bu yüzden atanmaması gerekiyor. Şu anki issue AYS-661 ve onun içerisinde yazanları yapmamız gerekiyor